docs/tutorials/user-transpiled-circuits.ipynb

```diff
@@ -1179,7 +1179,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Finally, we can plot the results from the device runs against the ideal distribution. For each measured string, the leftmost bar is the ideal distribution, the central bar is the circuit transpiled with `optimization_level=3`, and the rightmost bar is the circuit transpiled with `optimization_level=0`). \n",
+    "Finally, we can plot the results from the device runs against the ideal distribution. For each measured string, the rightmost bar is the ideal distribution, the bar second to the left is the circuit transpiled with `optimization_level=3`, and the leftmost bar is the circuit transpiled with `optimization_level=0`). \n",
     "\n",
     "You can see the results with `optimization_level=3` is closer to the ideal distribution due to the lower gate count, and `optimization_level=3 + dd` is even closer due to the dynamic decoupling we applied."
```

Comment:
```suggestion
    "Finally, we can plot the results from the device runs against the ideal distribution. You can see the results with `optimization_level=3` are closer to the ideal distribution due to the lower gate count, and `optimization_level=3 + dd` is even closer due to the dynamic decoupling we applied."
```

On reflection, I think this sentence is redundant as key legend is clear in the diagram.